lib/success.js

```diff
@@ -130,6 +138,15 @@ export default async function success(pluginConfig, context, { Octokit }) {
 
     await Promise.all(
       uniqBy([...prs, ...issues], "number").map(async (issue) => {
+        const canCommentOnIssue = successCommentCondition
+          ? template(successCommentCondition)({ ...context, issue })
+          : true;
+
+        if (!canCommentOnIssue) {
```

Comment:
What do you think of applying `.filter` before `.map` to avoid the iteration with `undefined` values?

Reply:
Sounds good to catch an early error cause.

but I assume that the `uniqBy()` operation already filters out any item that doesn't satisfy the conditions of the iteratee i.e. the value to specify uniqueness. 🤔